### ISSUE TITLE
Problem with `read_extxyz`

### ISSUE THREAD

**Issue by wcwitt** [JuliaMolSim/JuLIP.jl#163]
In a fresh directory with the PRX silicon database (https://www.repository.cam.ac.uk/handle/1810/317974), the following fails
```
julia -e 'using Pkg; Pkg.activate("."); Pkg.add("JuLIP"); using JuLIP; read_extxyz("gp_iter6_sparse9k.xml.xyz")'
```
with
```
ERROR: KeyError: key "pbc" not found
Stacktrace:
 [1] getindex
   @ ./dict.jl:498 [inlined]
 [2] Atoms(D::Dict{String, Any})
   @ JuLIP ~/.julia/packages/JuLIP/lMIUi/src/atoms.jl:342
 [3] read_dict(#unused#::Val{:JuLIP_Atoms}, D::Dict{String, Any})
   @ JuLIP ~/.julia/packages/JuLIP/lMIUi/src/atoms.jl:348
 [4] read_dict(D::Dict{String, Any})
   @ ACEbase.FIO ~/.julia/packages/ACEbase/5cUVy/src/fio.jl:69
 [5] _extxyz_dict_to_atoms(dict::Dict{String, Any})
   @ JuLIP ~/.julia/packages/JuLIP/lMIUi/src/atoms.jl:459
 [6] read_extxyz(::String; kwargs::Base.Pairs{Symbol, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
   @ JuLIP ~/.julia/packages/JuLIP/lMIUi/src/atoms.jl:486
 [7] read_extxyz(::String)
   @ JuLIP ~/.julia/packages/JuLIP/lMIUi/src/atoms.jl:483
 [8] top-level scope
   @ none:1
```

**Comment by wcwitt** [comment]
I vaguely remembered seeing this before, and just found the old issue. So maybe it's just a matter of moving the fix to a new branch/version?

https://github.com/JuliaMolSim/JuLIP.jl/issues/151

**Comment by cortner** [comment]
ok - so the problem is we fixed this in an old branch  but not in main/master?

**Comment by cortner** [comment]
closed by #164
